### ISSUE TITLE
[BUG] Game Exited on Chromebook

### ISSUE THREAD

**Issue by 09beckerboy** [PojavLauncherTeam/PojavLauncher#3000]
### Describe the bug

Game Exited

### The log file and images/videos

[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/8364997/latestlog.txt)

https://user-images.githubusercontent.com/80997423/160455195-316b615f-e0e5-4853-9fb2-350fdd12737a.mp4

### Steps To Reproduce

```markdown
1. Start PojavLauncher
2. Click play on any version
3. Controls show up
4. Crashes with error "Game exited"
```

### Expected Behavior

Game launching with no problem

### Platform

```markdown
- Device model: Samsung Chromebook XE501C13-SO1US
- CPU architecture: x86 (I think)
- Android version: Chrome OS Version 97.0.4692.102
- PojavLauncher version: crocus-v3_openjdk
```

**Comment by wellsilver** [comment]
enable linux tools and download minecraft for linux, if it doesnt open run "sudo apt install secretlib" 
android and chromeos are different and even though there is cross=platform support it isnt perfect

tl;dr I diagnose skill issue

**Comment by 09beckerboy** [comment]
> enable linux tools and download minecraft for linux, if it doesnt open run "sudo apt install secretlib" android and chromeos are different and even though there is cross=platform support it isnt perfect
> 
> tl;dr I diagnose skill issue

I can't linux isn't supported on my chromebook, but thanks anyways

**Comment by GAOCentral** [comment]
https://www.youtube.com/watch?v=A6eSpOJqPAI
this should fix it

**Comment by ghost** [comment]
> enable linux tools and download minecraft for linux, if it doesnt open run "sudo apt install secretlib" android and chromeos are different and even though there is cross=platform support it isnt perfect
> 
> tl;dr I diagnose skill issue
*************@penguin:~$ sudo apt install secretlib
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package secretlib

**Comment by Mathias-Boulay** [comment]
hopefully fixed on the release candidate build.
